apps/crc_interactive.py

```diff
@@ -97,6 +98,11 @@ def _validate_arguments(self, args: Namespace) -> None:
         if args.mpi and (not args.partition == 'compbio') and args.num_nodes < self.min_mpi_nodes:
             self.error(f'You must use at least {self.min_mpi_nodes} nodes when using the MPI cluster')
 
+        # Check the minimum number of cores are requested for mpi
+        if args.mpi and args.num_cores < self.min_mpi_cores[args.partition]:
```

Comment:
This check will break if/when a new MPI partition is ever added. Instead of`self.min_mpi_cores[args.partition]`, I suggest using something like `self.min_mpi_cores.get(args.partition, SOME_DEFAULT)`.

Reply:
Thanks for the heads up, I switched it to use a default of 28 cores as we don't currently have any mpi nodes with a lower core count than that. Supplying `scavenger` (not in the dictionary) works as expected:

```
(wrappers_deploy) [nlc60@moss wrappers] mpi_cores : crc-interactive -m -p scavenger -n 2 -c 1
You must request at least 28 cores per node when using the MPI cluster scavenger partition
```